src/vorta/views/utils.py

```diff
@@ -3,7 +3,7 @@
 from vorta.utils import get_asset, uses_dark_mode
 
 
-def get_colored_icon(icon_name):
+def get_colored_icon(icon_name, scaled_height=120, return_qpixmap=False):
```

Comment:
You now scale to 120, where previously the default was 128. Has that a reason?

Reply:
Sorry - typo. Fix pushed :)